nservicebus/upgrades/asp-6to1.md

```diff
@@ -10,7 +10,7 @@ related:
 - nservicebus/upgrades/asp-saga-deduplication
 ---
 
-WARNING: Upgrades from NServiceBus.Azure v6.2.3 or lower will need to apply the [saga de-duplication patch](/nservicebus/upgrades/asp-saga-deduplication.md) before completing the remainder of these upgrade steps.
+WARNING: Upgrades from NServiceBus.Azure v6.2.3 or lower will need to apply the [saga de-duplication patch](/nservicebus/upgrades/asp-saga-deduplication.md) and the [saga index patch](/nservicebus/upgrades/asp-saga-pruning.md) before completing the remainder of these upgrade steps.
```

Comment:
It doesn't but I've updated the text to suggest the order to keep in line with our "do each upgrade in sequence" policy